symphony-sample/packer-templates/scripts/inventory

```diff
@@ -0,0 +1,4 @@
+[all:vars]
+ansible_connection=ssh
+ansible_user=root
+ansible_ssh_pass=p@ssw0rd
```

Comment:
Can we provide the ssh_pass through the command line, and have it use the packer variable? Otherwise, it's one more place the user has to update, to use a non-default pw

Reply:
Removed the inventory file. ansible now uses the ssh_username and ssh_pass from the packer builder step